### PR TITLE
Add cep.cr to the list

### PR DESCRIPTION
**Pull request by vnbrs**

### CODE REVIEW

**Comment by vnbrs** [comment]
@RX14 Thank you! I'm gonna do that. Those are my first lines of code in Crystal :P

**Comment by vnbrs** [comment]
@RX14  https://github.com/vnbrs/cep.cr/pull/1/files

**Comment by RX14** [comment]
@vnbrs I can't comment on your PR but how about returning just `response.body` from `Service.get` instead of parsing to json then back again

File: README.md [hunk 1]
```diff
@@ -462,6 +462,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [awscr-s3](https://github.com/taylorfinnell/awscr-s3) - AWS S3 interface
  * [awscr-signer](https://github.com/taylorfinnell/awscr-signer) - Sign HTTP::Request objects and generate presigned post forms
  * [bugsnag.cr](https://github.com/gewo/bugsnag.cr) - Bugsnag exception notifier and sidekiq middleware
+ * [cep.cr](https://github.com/vnbrs/cep.cr) - A library to fetch Brazilian post codes/CEPs easily.
```
Comment (veelenga) [inline README.md]: Remove a full stop at the end (i.e. `.`) for consistency.

**Comment by vnbrs** [comment]
@hugoabonizio Hello from Curitiba 🇧🇷!

I've just done the changes. I've read this performance guide, which is awesome, thank you!

**Comment by hugoabonizio** [comment]
Since this shard was merged into `burocracia.cr` (vnbrs/burocracia.cr#7), let's keep only one of them.

**Pull request closed without merging**